client/app/main/visualization/visualization.controller.js

```diff
@@ -2,97 +2,115 @@
 
 function VisualizationCtrl($scope, $timeout, Sidebar, Visualization) {
   $scope.sidebar = Sidebar;
-  $scope.statType = 'Cumulative';
-  $scope.truancyType = 'Absences';
+  $scope.studentType = 'nonCfa';
+  $scope.data = { 
+    cfa: { arca: 0, ca: 0, other: 0 }, 
+    nonCfa: { arca: 0, ca: 0, other: 0 } 
+  };
   $scope.chartConfig = {
     options: {
       chart: {
-        zoomType: 'x',
-        type: 'line'
-      }
-    },
-    xAxis: {
-      type: 'datetime',
-      title: {
-        text: 'Date'
-      }
-    },
-    yAxis: {
+        plotBackgroundColor: null,
+        plotBorderWidth: null,
+        plotShadow: false,
+        type: 'pie'
+      },
       title: {
-        text: $scope.truancyType
+        text: 'ARCA vs CA'
       },
-      allowDecimals: $scope.truancyType === 'Average'
-    },
-    title: {
-      text: $scope.statType + ' ' + $scope.truancyType + ' ' + 'Trends'
+      tooltip: {
+        pointFormat: '{series.name}: <b>{point.percentage:.1f}%</b>'
+      },
+      plotOptions: {
+        pie: {
+          allowPointSelect: true,
+          cursor: 'pointer',
+          dataLabels: {
+            enabled: false,
+            format: '<b>{point.name}</b>: {point.percentage:.1f} %',
+            style: {
```

Comment:
This looks to be the default. Using "Highcharts" breaks the build because it is not an allowed global.  lets remove this style property.